ansible_collections/arista/avd/plugins/action/anta_workflow.py

```diff
@@ -257,17 +270,17 @@ def build_reports(batch_results: Iterator[ResultManager], report_settings: dict[
         for result in manager.results:
             result_manager.add(result)
 
-    # Filter the results based on the hide_statuses if provided
-    if hide_statuses:
-        filtered_result_manager = result_manager.filter(hide=set(hide_statuses))
+    # Filter the results based on the exclude_statuses if provided
+    if exclude_statuses:
+        filtered_result_manager = result_manager.filter(hide=set(exclude_statuses))
         if not filtered_result_manager.results:
-            msg = f"The report is empty because all results were hidden by the provided status filters: {', '.join(hide_statuses)}"
+            msg = f"The report is empty because all results were hidden by the provided status filters: {', '.join(exclude_statuses)}"
             LOGGER.warning(msg)
     else:
         filtered_result_manager = result_manager
 
     # Sort the result manager
-    filtered_result_manager.sort(sort_by=["name", "categories", "test", "description", "result", "custom_field"])
+    filtered_result_manager = sort_results(filtered_result_manager, sort_by_status_priority, sort_test_results)
```

Comment:
No need to return the same object.
```suggestion
    sort_results(filtered_result_manager, sort_by_status_priority, sort_test_results)
```

Reply:
Updated